src/pybind/mgr/dashboard/frontend/src/app/ceph/rgw/rgw-bucket-form/rgw-bucket-form.component.html

```diff
@@ -279,6 +279,112 @@
           </div>
         </fieldset>
 
+        <fieldset>
+          <legend class="cd-header"
+                  i18n>Security</legend>
+
+          <!-- Locking enabled -->
+          <div class="form-group row">
+            <div class="cd-col-form-offset">
+              <div class="custom-control custom-checkbox">
+                <input class="custom-control-input"
+                       id="encryption_enabled"
+                       name="encryption_enabled"
+                       formControlName="encryption_enabled"
+                       type="checkbox"
+                       [attr.disabled]="!kmsVaultConfig && !s3VaultConfig ? true : null">
+                <label class="custom-control-label"
+                       for="encryption_enabled"
+                       i18n>Encryption</label>
+                <cd-helper>
+                  <span i18n>Enables encryption for the objects in the bucket.
+                     To enable encryption on a bucket you need to set the configuration values for SSE-S3 or SSE-KMS.
+                     To set the configuration values <a href="#/rgw/bucket/create" (click)="openConfigModal()">Click here</a></span>
```

Comment:
If the encryption is enabled, then I guess we don't need to show this part of the help text. We can conditionally disable this then! WDYT?

Reply:
The user can change the config values whenever required.